### PR TITLE
Precision

### PR DESCRIPTION
**Pull request by jalelmahjoubi**

### CODE REVIEW

**Comment by EmmanuelDemey** [comment]
@jalelmahjoubi what is this new .history folder ?

**Comment by jalelmahjoubi** [comment]
> @jalelmahjoubi what is this new .history folder ?

File: src/model/transformations/codes-list.js [hunk 1]
```diff
@@ -36,8 +38,9 @@ export function getcodelistwithclarification(remote, variableclarification) {
   remote.forEach(codelist => {
     variableclarification.forEach( clarif => {
       if (clarif.codelistid === codelist.id){
-          codelist.Code[clarif.position] = {
-            ...codelist.Code[clarif.position], 
+      const index = codelist.Code.findIndex(code => code.Value == clarif.position)
```
Comment (EmmanuelDemey) [inline src/model/transformations/codes-list.js:41]: wrong indentation

File: src/model/transformations/collected-variable.js [hunk 2]
```diff
@@ -3,13 +3,24 @@ import {
...
+import { element } from 'prop-types';
 const { COLLECTED } = VARIABLES_TYPES;
 
 export function remoteToStore(
   remote = [],
   responsesByVariable,
   codesListsStore,
+  variableclarification,
 ) {
+  remote.map(variable => {
```
Comment (EmmanuelDemey) [inline src/model/transformations/collected-variable.js:15]: a .map function should return something. 
If it is not the case, you should prefer .forEach

File: src/widgets/codes-lists/components/codes-lists-input-code.jsx [hunk 3]
```diff
@@ -82,7 +82,7 @@ class CodesListInputCode extends ComponentWithValidation {
         change(formName, `${path}precisionlabel`, code.precisionlabel);
       }
       else if(precisionShow) { 
-        change(formName, `${path}precisionlabel`, `${Question}${code.value}CL label`);
+        change(formName, `${path}precisionlabel`, `Préciser :`);
```
Comment (EmmanuelDemey) [inline src/widgets/codes-lists/components/codes-lists-input-code.jsx:85]: I think the "Préciser :" string should come from the dictionary.

**Pull request merged**